dynatrace-aws-logs.sh

```diff
@@ -242,7 +242,7 @@ EOF
 
   set -e
 
-  echo "Deploying stack $STACK_NAME"
+  echo "Deploying stack $STACK_NAME. This might take about 10 minutes if you chose included AG option, or about 5 minutes if you chose existing AG option."
```

Comment:
? Is this a consequence of changes with subnets/VPCs in this PR? Because that would seem like a penalty on existingAG setup for some improvement for the other (newAG) one and to me at least a warning sign to reconsider what we're doing. I never noticed it taking 5 min so far, but certainly I wasn't looking, so asking.

Reply:
I don't feel it ever took 10 min with included AG either. I also believe the option is referred to as "new AG" in the docs most of the time. One more minor thing is I think I would say "This will take about.." since you still have uncertainty by saying "about" - I don't see what additional uncertainty the "might" would refer to.

Reply:
For me it's about it *might* take less:P Good point about the naming.

Reply:
No, nothing changes with regard to time (maybe a little, new subnet and NatGateway).
I just checked and this deployment does take 5 minutes for Cloudformation (with included AG). 

I just pushed an update, please check if you like it. I put "up to 10 minutes" without separating scenarios. This has only one goal - if user sees "waiting for stack update/create" for 2nd minute, he does not panic but sees this info and keeps waiting. It does not hurt to give longer estimate here, I think AWS can also surprise sometimes and have its blips. 

For the naming, we are rebranding this in the docs right now (as discussed within team some time ago). There will be includedAG and existingAG. It's 95% sure at the moment, I am consulting this right now with Jarek Orlowski. He proposed "CloudFormation-included" and also "stack-included" etc.

Reply:
Ok, talked with Jarek and we settled for stack-included after language consultations with Malcolm